runtime/doc/options.txt

```diff
@@ -4872,6 +4872,21 @@ A jump table for the options with a short description can be found at |Q_op|.
 <	Replace the ';' with a ':' or whatever separator is used.  Note that
 	this doesn't work when $INCL contains a comma or white space.
 
+							   *'plugindir'*
+'plugindir' 'pldir'	string (default "plugged")
```

Comment:
Vim-pi is Vim Plugin Index. I have not seen it recognized as abbreviation for `plugin`, not sure why exactly Vim help uses `pi`.